### PR TITLE
FEATURE 2-Admin Login added

### PR DESCRIPTION
**Pull request by mich2635**
##### Feature 2

- [x] Admin Login Added
- [x] Admin Login Validator Added
- [x] Admin Login jsp file added
- [x] Admin Login Junit Added

### CODE REVIEW

File: src/main/java/in/micheal/validator/AdminLoginValidator.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	private AdminLoginValidator() {
+		// default constructor
+	}
+
+	public static boolean loginValidator(long adminId, String password) {
+		boolean confirmation = false;
+		for (UserDetails user : AllDetailsDB.userDetails) {
+			if (user.getUserId() == adminId && user.getAdminPassword().equals(password)) {
+				confirmation = true;
+			}
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/validator/AdminLoginValidator.java:18]: Add break when the searched item is found.

**Review by ghost — APPROVED** [review]

**Comment by csys-fresher-batch-2021** [comment]
I have reviewed the changes. It looks good for me .

File: src/main/webapp/AdminView.jsp [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
```
Comment (Christykumar) [inline src/main/webapp/AdminView.jsp:7]: Insert some valid title

File: src/main/webapp/header.jsp [hunk 3]
```diff
@@ -2,6 +2,9 @@
 <link rel="stylesheet" href="assets/css/fontawesome.min.css">
 <link rel="stylesheet" href="assets/css/style.css">
 
+
+
+		
 <header>
 <nav class="navbar navbar-expand-sm navbar-dark bg-dark">
   <a class="navbar-brand" href="#">MyApp</a>
```
Comment (Christykumar) [inline src/main/webapp/header.jsp:10]: change to your app name

**Pull request closed without merging**